connector/connect/common/pom.xml

```diff
@@ -156,6 +156,43 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Shade all GRPC / Guava / Protobuf dependencies of this build -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <configuration>
+                    <shadedArtifactAttached>false</shadedArtifactAttached>
+                    <artifactSet>
```

Comment:
If `connect-common.jar` will not be used independently, I guess it can be used only by configuring the `relocations` rule(same as `connect` module), the `artifactSet` doesn't need to be configured.

Reply:
I will test this guess manually

Reply:
Thanks for reviewing this thoroughly, @LuciferYang

Reply:
> I will test this guess manually

After manual check, configuring
```
<artifactSet>
  <includes>
    <include>com.google.guava:*</include>
  </includes>
</artifactSet>
```
is redundant

Reply:
Thank you. Would you like to create a followup if you don't mind?